lib/transactions.js

```diff
@@ -0,0 +1,93 @@
+import { saveAs } from 'file-saver';
+import { round } from 'lodash';
+
+import { get as fetch } from './api';
+import { formatCurrency } from './currency-utils';
+import { toIsoDateStr } from './date-utils';
+import { collectiveInvoiceURL, invoiceServiceURL, transactionInvoiceURL } from './url_helpers';
+
+export const formatFee = (value, totalAmount, currency, name, showPercent = true) => {
+  if (!value || !totalAmount) {
+    return '';
+  }
+  const v =
+    value < 0 ? ` - ${formatCurrency(Math.abs(value), currency)}` : ` + ${formatCurrency(Math.abs(value), currency)}`;
+  if (showPercent) {
+    const percentage = round((value / totalAmount) * 100, 2);
+    return `${v} (${percentage}% ${name})`;
+  } else {
+    return `${v} (${name})`;
+  }
+};
+
+export const renderDetailsString = ({
+  amount: _amount,
+  platformFee,
+  hostFee,
+  paymentProcessorFee,
+  netAmount: _netAmount,
+  isCredit,
+}) => {
+  const [amount, netAmount] = !isCredit && _netAmount ? [_netAmount, _amount] : [_amount, _netAmount];
```

Comment:
How do we explain this behavior? Does it make also sense when looking at the reverse transactions? (The Organization who contributed money, the Individual who submitted an expense).

Maybe instead of looking at `isCredit` we want to display something different regarding if it's an expense or a financial contribution.

Reply:
Why are we not displaying details for Expenses? We used to:

<img width="1101" alt="Screenshot 2020-08-01 at 00 19 25" src="https://user-images.githubusercontent.com/806/89081878-ad06af00-d38c-11ea-920d-14d863fad4df.png">

Reply:
> I'm just swapping amount and net amount because these are flipped in our DB.

I would like us to take this as an opportunity to understand why the data is stored like that in the DB. Challenge it, and update how it's stored if it doesn't make sense.

Should not block this PR but let's discuss next week.